package.json

```diff
@@ -124,7 +125,8 @@
         "typedoc-plugin-coverage": "^3.0.0",
         "typedoc-plugin-mdn-links": "^4.0.0",
         "typedoc-plugin-missing-exports": "^3.0.0",
-        "typescript": "^5.4.2"
+        "typescript": "^5.4.2",
+        "wait-for-expect": "^3.0.2"
```

Comment:
I think jest's `waitFor` is what you want here, this hasn't been updated in 5 years.

Reply:
this would require me to import add testing-library-dom?
I think right now we do not have `waitFor` in the js-sdk?